via.js

```diff
@@ -269,6 +269,24 @@ define(['./utils'], function (utils) {
                             temp &= 0x7f;
                             temp |= (self.t1_pb7 << 7);
                         }
+                        
+                        var buttons = this.getJoysticks();
+
+                        // clear PB4 and PB5
+                        temp = temp & 0xCF // 11001111
```

Comment:
There's a number of missing semicolons: the `lint` check is picking these up too!